src/FSharp.Data.GraphQL.Server/Execution.fs

```diff
@@ -228,19 +222,6 @@ let private createFieldContext objdef argDefs ctx (info: ExecutionInfo) =
       Args = args
       Variables = ctx.Variables }         
                 
-
-let private optionCast (value: obj) =
```

Comment:
I took out those functions and put them on the Shared library, because I wanted to use them on another piece of code for this issue.